docs.markdown

```diff
@@ -1291,416 +1399,580 @@ smooks.filterSource(new StreamSource(csvReader), result);
 
-#### Required Values & Truncation
+```xml
+<?xml version="1.0"?>
+<smooks-resource-list xmlns="https://www.smooks.org/xsd/smooks-1.2.xsd"
+                      xmlns:dfdl="https://www.smooks.org/xsd/smooks/dfdl-1.0.xsd">
 
-*   **required**: \<field>, \<component> and \<sub-component> configurations support a "required" attribute, which flags that <field>, <component> or <sub-component> as requiring a value.
+    ...
 
-By default, values are not required (fields, components and sub-components).
+    <dfdl:unparser schemaURI="/csv.dfdl.xsd" unparseOnElement="file"/>
 
-*   **truncatable**: \<segment>, \<field> and \<component> configurations support a "truncatable" attribute. For a segment, this means that parser errors will not be generated when that segment does not specify trailing fields that are not "required" (see "required" attribute above). Likewise for fields/components and components/sub-components.
+</smooks-resource-list>
+```
 
-By default, segments, fields, and components are not truncatable.
+The DFDL cartridge supports variables, on disk caching, and trace debugging. Consult the [XSD documentation](/xsd/smooks/dfdl-1.0.xsd) for further
+information.
 
-So, a \<field>, \<component> and \<sub-component> can be present in a message in one of the following states:
 
-1.  Present with a value _(required="true")_
-2.  Present without a value _(required="false")_
-3.  Not Present _(required="false" and truncatable="true")_
+EDI Cartridge
+-------------
 
-### Imports
+### Maven Coordinates
 
-Many message groups use the same segment definitions. Being able to define these segments once and import them into a top level configuration saves on a lot of duplication. A simple configuration demonstrating the import feature would be as follows:
 ```xml
-<?xml version="1.0" encoding="UTF-8"?>  
-<medi:edimap xmlns:medi="https://www.smooks.org/schema/edi-message-mapping-1.7.xsd">  
-   
-    <medi:import truncatableSegments="true" truncatableFields="true" truncatableComponents="true" resource="example/edi-segment-definition.xml" namespace="def"/>  
-   
-    <medi:description name="DVD Order" version="1.0"/>  
-   
-    <medi:delimiters segment="&#10;" field="*" component="^" sub-component="~" escape="?"/>  
-   
-    <medi:segments xmltag="Order">  
-        <medi:segment minOccurs="0" maxOccurs="1" segref="def:HDR" segcode="HDR" xmltag="header"/>  
-        <medi:segment minOccurs="0" maxOccurs="1" segref="def:CUS" segcode="CUS" xmltag="customer-details"/>  
-        <medi:segment minOccurs="0" maxOccurs="-1" segref="def:ORD" segcode="ORD" xmltag="order-item"/>  
-    </medi:segments>  
-   
-</medi:edimap>
-```
-The configuration-example above demonstrates the use of import that were introduced in Smooks v1.1, where single segments or segments containing childsegments can be separated into a separate file for better reuse in the future.
+<dependency>
+    <groupId>org.smooks.cartridges.edi</groupId>
```

Comment:
I see that the the `groupId` for `smooks-dfdl-cartridge` is `org.smooks.cartridges` while for `smooks-edi-cartridge` it's `org.smooks.cartridges.edi`.

Are there multiple EDI cartridges?

If not, does the `.edi` suffix on the `groupId` not seem a bit inconsistent with the other cartridges, as well as being a bit redundant.

Not a biggie and not worth changing if it's a lot of bother.